cmd/kubeadm/.import-restrictions

```diff
@@ -63,7 +63,6 @@
 				"k8s.io/kubernetes/pkg/controller",
 				"k8s.io/kubernetes/pkg/features",
 				"k8s.io/kubernetes/pkg/fieldpath",
-				"k8s.io/kubernetes/pkg/kubeapiserver/authorizer/modes",
```

Comment:
@liggitt Any chance for us to be able to move the modes package somewhere in staging (such as `k8s.io/apiserver/pkg/authorization/authorizer/modes`)?

/hold

Reply:
the modes don't apply to all apiservers, just kube-apiserver. that doesn't yet have a config package outside k/k